packages/core/src/components/drawer/drawer.tsx

```diff
@@ -146,6 +146,10 @@ export class Drawer {
     });
   }
 
+  componentDidRender() {
```

Comment:
Please check  https://stenciljs.com/docs/component-lifecycle#componentdidrender `componentDidRender` looks not like a good place to do this.